.github/workflows/build-container.yml

```
@@ -42,12 +42,24 @@ jobs:
         tags=()
         for registry in hub.opensciencegrid.org docker.io; do
           tags+=( $registry/$docker_repo:$OSG_RELEASE-$BASE_YUM_REPO-el$EL_VER )
-          tags+=( $registry/$docker_repo:$OSG_RELEASE-el$EL_VER )
-          tags+=( $registry/$docker_repo:$OSG_RELEASE-$BASE_YUM_REPO )
-          tags+=( $registry/$docker_repo:$OSG_RELEASE )
-          tags+=( $registry/$docker_repo:$BASE_YUM_REPO )
-          if [[ $BASE_YUM_REPO == "release" ]]; then 
-            tags+=( $registry/$docker_repo:latest )
+          if [[ $EL_VER == '8' ]]; then
+            tags+=( $registry/$docker_repo:$OSG_RELEASE-el$EL_VER )
+            tags+=( $registry/$docker_repo:$OSG_RELEASE-$BASE_YUM_REPO )
+            if [[ $BASE_YUM_REPO == 'release' ]]; then
+              tags+=( $registry/$docker_repo:$OSG_RELEASE )
+              if [[ $OSG_RELEASE == '3.6' ]]; then
+                tags+=( $registry/$docker_repo:latest )
+                tags+=( $registry/$docker_repo:$BASE_YUM_REPO )
+              fi
+            else
+              if [[ $OSG_RELEASE == '3.6' ]]; then
+                tags+=( $registry/$docker_repo:$BASE_YUM_REPO )
+              fi
+            fi
+          else
+            if [[$BASE_YUM_REPO == 'release' ]]; then
+              tags+=( $registry/$docker_repo:$OSG_RELEASE-el$EL_VER )
+            fi
           fi
```

Bash's `elif` can be used here as an "else if":

```suggestion
          elif [[$BASE_YUM_REPO == 'release' ]]; then
            tags+=( $registry/$docker_repo:$OSG_RELEASE-el$EL_VER )
          fi
```